conf/machine/jetson-orin-nano-devkit-nvme.conf

```diff
@@ -4,8 +4,11 @@
 
 MACHINEOVERRIDES =. "jetson-orin-nano-devkit:"
 TNSPEC_BOOTDEV ?= "nvme0n1p1"
-PARTITION_LAYOUT_TEMPLATE ?= "flash_t234_qspi.xml"
-PARTITION_LAYOUT_EXTERNAL ?= "flash_l4t_external.xml"
+PARTITION_LAYOUT_TEMPLATE_DEFAULT ?= "flash_t234_qspi.xml"
+PARTITION_LAYOUT_TEMPLATE_DEFAULT_SUPPORTS_REDUNDANT ?= "1"
```

Comment:
Wouldn't hurt to add a comment here to explain why this is being set.  I think I get it, but it's not necessarily intuitive.

Reply:
I put a small dissertation about this in https://github.com/OE4T/meta-tegra/pull/1428/commits/e0e04e85d4acd1c34b55620d749d3f5180b14ac7 but can move some of it here, or maybe in the weak definition of `PARTITION_LAYOUT_TEMPLATE_DEFAULT_SUPPORTS_REDUNDANT`

Reply:
Wiki page would be OK too.  Documenting it somewhere other than the commit history is helpful.

Reply:
I've started a wiki page at https://github.com/OE4T/meta-tegra/wiki/Rendundant-Rootfs-A-B-Partition-Support